source/blog/2016-07-20-vagrant-service-manager-install-cli.html.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Download client binaries through vagrant-service-manager in ADB
+author: budhram
+date: 2016-07-20 09:56 UTC
+tags: atomic-developer-bundle, vagrant-service-manager
+published: true
+comments: true
+---
+
+The [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) plugin is designed to enable easier access to the features and services provided by the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). More information can be found in the README  of 'vagrant-service-manager' repo.
+
+Here, I am going to show the new prominent command `install-cli` which got released with version 1.2.0. This command installs the client binary for any of the services provided by ADB. Currently, 'docker' and 'openshift' are supported.
```

Comment:
This blog is about the new feature in vagrant-service-manger i.e.  `install-cli` command. "install-cli" got released with version vagrant-service-manager version 1.2.0. This command installs the client binary for any of the services provided by ADB. Currently, 'docker' and 'openshift' are supported.